From 738c5abc1f2b628f5d9a3ffd273ea85aaa45b38a Mon Sep 17 00:00:00 2001
From: insilications <boboniboni@gmail.com>
Date: Wed, 5 Aug 2020 05:58:10 -0300
Subject: [PATCH 1/3] Modify token tests to reflect the mockroot permissions

The paths used in some of the tests don't have the permissions
expected and so tests will fail. Modify the paths to ones whose
permissions are what the tests expect (in the Clear Linux build
environment the mock group has write permissions to /).
---
 trust/test-token.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/trust/test-token.c b/trust/test-token.c
index 4ff1f2f..17c14c9 100644
--- a/trust/test-token.c
+++ b/trust/test-token.c
@@ -240,7 +240,7 @@ test_not_writable (void)
 
 #ifdef OS_UNIX
 	if (getuid () != 0) {
-		token = p11_token_new (333, "/", "Label", P11_TOKEN_FLAG_NONE);
+        token = p11_token_new (333, "/root", "Label", P11_TOKEN_FLAG_NONE);
 		assert (!p11_token_is_writable (token));
 		p11_token_free (token);
 	}
@@ -252,7 +252,7 @@ test_not_writable (void)
 
 #ifdef OS_UNIX
 	if (getuid () != 0) {
-		token = p11_token_new (333, "/non-existant", "Label", P11_TOKEN_FLAG_NONE);
+        token = p11_token_new (333, "/usr/non-existant", "Label", P11_TOKEN_FLAG_NONE);
 		assert (!p11_token_is_writable (token));
 		p11_token_free (token);
 	}
-- 
2.27.0

